.github/workflows/build_deploy.yml

```
@@ -21,9 +21,9 @@ jobs:
     strategy:
       matrix:
         include:
-         - os: ubuntu-18.04
+         - os: ubuntu-22.04
```

```suggestion
         - os: ubuntu-latest
```

then it is the same as all of our other builds?